### DISCUSSION TITLE
Minor tick direction

### DISCUSSION THREAD

**Discussion by ygk2zhang** [has2k1/plotnine#876] [Q&A]
Hi,

I would like to have minor ticks in plots pointing **in**. I don't know how to do it.
I can get them enabled in **theme** 
`+ theme( axis_ticks_minor=element_line(), axis_ticks_direction='in' )+scale_x_continuous(breaks=[ 0,5, 10, 15, 20, ], limits=[0,20], minor_breaks=4)`
but the result is the following image. 
![image_2024-09-27_104756054](https://github.com/user-attachments/assets/e13268e7-2583-4f83-b238-ebda4006c2e3)

**Comment by has2k1** [comment]
Transferred this to issues. It is a bug.
